app/assets/v2/css/activity_stream.css

```diff
@@ -79,6 +79,10 @@ a .sup {
 .action.open {
   background-color: #eeeeee;
 }
+.emoji-container,
+.comment-area{
```

Comment:
Do we need set the `.comment-area` to `top:0` ?

Reply:
No we didn't, I have since removed the rule and a few others that weren't necessary within inline css.